### PR TITLE
git-hound 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by ezekg**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Follow up to https://github.com/Homebrew/homebrew-core/pull/61757 because I force pushed and it somehow broke my branch ref.

### CODE REVIEW

**Comment by ezekg** [comment]
Build is failing, but I was getting these errors earlier too while making this PR so it may be a GitHub/network issue?

```
==> git -C /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core fetch origin +refs/heads/master
==> FAILED
fatal: unable to access 'https://github.com/Homebrew/homebrew-core/': transfer closed with outstanding read data remaining
```

cc @dawidd6, since you were reviewing my previous PR. 🙂

**Comment by dawidd6** [comment]
Yes, it's an issue with our CI. Happens randomly.

File: Formula/git-hound.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  desc "Git plugin that prevents sensitive data from being committed"
+  homepage "https://github.com/ezekg/git-hound"
+  url "https://github.com/ezekg/git-hound/archive/1.0.0.tar.gz"
+  sha256 "32f79f470c790db068a23fd68e9763b3bedc84309a281b4c99b941d4f33f5763"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```
Comment (SMillerDev) [inline Formula/git-hound.rb:11]: Does the project not use go modules?

Comment (ezekg) [reply Formula/git-hound.rb:11]: I just updated it to use Go modules yesterday, before making this PR. Is this line not required? Sorry — first time making a PR here so I kind of based my install on a couple other Go formulas like Caddy.

Comment (SMillerDev) [reply Formula/git-hound.rb:11]: You probably only need something like telegraf: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/telegraf.rb

Comment (ezekg) [reply Formula/git-hound.rb]: Updated 👍

**Review by dawidd6 — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @ezekg  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**